client/my-sites/themes/single-site.jsx

```diff
@@ -37,30 +37,31 @@ const ThemesSingleSite = ( props ) => {
 		{ analyticsPath, analyticsPageTitle, isJetpack, translate } = props,
 		jetpackEnabled = config.isEnabled( 'manage/themes-jetpack' );
 
-	if ( isJetpack && jetpackEnabled && ! site.hasJetpackThemes ) {
-		return <JetpackUpgradeMessage site={ site } />;
-	}
-
-	if ( isJetpack && jetpackEnabled && ! site.canManage() ) {
-		return <JetpackManageDisabledMessage site={ site } />;
-	}
-
-	if ( isJetpack && ! jetpackEnabled ) {
-		return (
-			<Main className="themes">
-				<PageViewTracker path={ analyticsPath } title={ analyticsPageTitle }/>
-				<SidebarNavigation />
-				<CurrentTheme
-					site={ site }
-					canCustomize={ site && site.isCustomizable() } />
-				<EmptyContent title={ translate( 'Changing Themes?' ) }
-					line={ translate( 'Use your site theme browser to manage themes.' ) }
-					action={ translate( 'Open Site Theme Browser' ) }
-					actionURL={ site.options.admin_url + 'themes.php' }
-					actionTarget="_blank"
-					illustration="/calypso/images/drake/drake-jetpack.svg" />
-			</Main>
-		);
+	if ( isJetpack ) {
```

Comment:
Sorry to comment on in-progress work. Ignore this if it's not relevant. I think that this extra indentation is not actually easier to read. I think there's a guideline somewhere that goes into this in more detail, although I can't find it ATM. I suggest that if the repetition of `if ( isJetpack...` seems over-verbose, we use something like this instead, which could contain the whole of this block:

```js
if ( isJetPack ) {
  return this.renderJetpack( site );
}
...
renderJetpack( site ) {
  const jetpackEnabled = config.isEnabled( 'manage/themes-jetpack' );
  if ( ! jetpackEnabled ) {
    return ...
  }
  if ( ! site.hasJetpackThemes ) {
    return ...
  }
  if ( ! site.canManage ) {
  }
}
```